proto/identity/api/v1/identity.proto

```diff
@@ -51,31 +50,35 @@ message PublishIdentityUpdateRequest {
   xmtp.identity.associations.IdentityUpdate identity_update = 1;
 }
 
+message PublishIdentityUpdateResponse {
+  string inbox_id = 1;
```

Comment:
Why is this needed? Is the client expected to validate this result?

Reply:
A `CreateIdentity` request doesn't contain the `inbox_id` explicitly, just the inputs. Just gives the clients some confirmation that the server came to the same conclusion they did.

Reply:
If someone wants to move a wallet between inbox ID's, can they publish a batch signature to revoke from one inbox ID and associate with another, and have it apply to both inbox ID's?

Reply:
Summary of discussion: it is possible to support multi-inbox identity updates in order to reduce the number of signatures a user needs to provide. However, this case should be rare and adds complexity, so we choose not to optimize for it.